discord/commands/core.py

```diff
@@ -1303,8 +1303,9 @@ def create_subgroup(
         """
 
         if self.parent is not None:
-            # TODO: Improve this error message
-            raise Exception("a subgroup cannot have a subgroup")
+            raise Exception(
+                "a slash command group can only have one level of subcommand groups"
```

Comment:
```suggestion
                "A slash command group can only have one level of subcommand groups"
```

Shouldn't we capitalize the first letter?